pkg/Library/NodeGraph/RecipeBuilder.js

```diff
@@ -104,7 +104,7 @@ buildParticleSpecs(node, nodeType, layout, {storeMap}) {
   const specs = {};
   const names = this.getParticleNames(nodeType) || [];
   for (const particleName of names) {
-    const container = layout?.[`${node.id}:Container`] || this.defaultContainer;
+    const container = layout?.[`${node.id}${nameDelim}Container`] || this.defaultContainer;
```

Comment:
i think it should be `this.nameDelim`, shouldn't it?

ps: sorry for messing the slot names ... again 🤦

Reply:
Yes. When using SES the member-references are projected into module scope as well as being on `this`, which allowed the 'no-this' syntax. Ray still does this sometimes I think because he hasn't grasped this detail.
I decided _not_ to do that, because I want to force people to name things on 'this' instead of passing objects. For example, a 'Decorator' shouldn't just be any old function, I want to make you supply a 'name' that is a method of 'this' (I think right now it supports both to be BC, I can't remember where I left it).

Reply:
This second method affords certain guarantees in the decorator contract, but I've paged out the details. Maybe we can ask Secops Ray to put it in a doc?

Reply:
Your last PRs were the big name cleanup, the fallout was minimal. Great work, really appreciate it.